MAUI/DataGrid/Column-types.md

```diff
@@ -521,6 +580,227 @@ dataGrid.Columns.Add(dateColumn);
 {% endhighlight %}
 {% endtabs %}
 
+## DataGridComboBoxColumn
+
+The [GridComboBoxColumn]() inherits all the properties of the [SfDataGrid.DataGridColumn](). It displays a list of items in the form of a `SfComboBox` as the content of a column. To enable or disable editing for a particular column, set the [DataGridColumn.AllowEditing]() property to true or false. When in editing mode, it displays a [SfComboBox]() element. The data source for the SfComboBox can be set using the [DataGridComboBoxColumn.ItemsSource]() property. The combobox column can be populated with data in the following ways:
+
+* Collection of primitive types
+* Collection of user-defined types (custom objects)
+
+![DataGrid with editing in comboBox column](Images\column-types\maui-datagrid-comboBox-column.png)
+
+### Collection of primitive types
+
+To display the collection of items in the comboBox drop down, create a [DataGridComboBoxColumn]() and set its [DataGridComboBoxColumn.ItemsSource]() property to a simple collection. 
+
+To load the `DataGridComboBoxColumn` with a simple string collection, follow the code example:
```

Comment:
To load the `DataGridComboBoxColumn` with a simple string collection, you can refer to the code example below:

Reply:
Had updated the Changes.